### ISSUE TITLE
Object-oriented checks in DT::E::Sunrise, contents of documentation [rt.cpan.org #7605]

### ISSUE THREAD

**Issue by jforget** [jforget/DateTime-Event-Sunrise#3]
Migrated from [rt.cpan.org#7605](https://rt.cpan.org/Ticket/Display.html?id=7605) (status was 'resolved')

Requestors:
* J2N-FORGET@orange.fr

Attachments:
* [SunRise.diff1](https://rt.cpan.org/Ticket/Attachment/111350/22424/SunRise.diff1)

From j2n-forget@orange.fr on 2004-09-10 05:35:44
:
```
It would be nice if you add a paragraph to your module's
documentation: what happens beyond the polar circle?
Does the module croak? Does the module return undef?
How should the module user code the script?

Other minor changes are included in the attached
diff file.

Jean Forget

```

**Comment by jforget** [comment]
Tuesday 2013-07-09, rt.cpan.org: fixed in 0.0502.

**Comment by jforget** [comment]
Patch provided on Friday 2004-09-10
[SunRise.diff1.txt](https://github.com/jforget/DateTime-Event-Sunrise/files/5869431/SunRise.diff1.txt)
